src/api/index.ts

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios'
+
+export const teamsApi = axios.create({
+  baseURL: 'http://localhost:3333',
```

Comment:
que tal usar env var?

```suggestion
  baseURL: process.env.REACT_APP_API_URL
```

Reply:
Eu pensei em fazer assim que eu fechar os testes e esilização dessa feature, logo depois iria fazer o deploy e colocaria a env